Demo/Fullscreen/BetaFeatureView/BetaFeatureDemoView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import FinniversKit
+
+final class BetaFeatureDemoView: UIView {
+    private lazy var betaFeatureView: BetaFeatureView = {
+        let view = BetaFeatureView(withAutoLayout: true)
+        view.configure(with: viewModel)
+        return view
+    }()
+
+    let viewModel = BetaFeatureViewModel(
+        iconImage: UIImage(named: "betaImageSearch"),
+        title: "Bildesøk på Torget",
+        description: "Test ut bildesøket vårt og finn lignende ting på Torget",
+        firstButtonTitle: "Ta nytt bilde",
+        secondButtonTitle: "Velg bilde..."
+    )
+
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func setup() {
+        addSubview(betaFeatureView)
+        layoutMargins = UIEdgeInsets(top: 0, leading: 16, bottom: 0, trailing: 16)
+        NSLayoutConstraint.activate([
```

Comment:
Can we use `fillInSuperviewLayoutMargins` here?

Reply:
not really, as in this case the bottom contraint is `lessThanOrEqualTo`